ios/Exponent/Versioned/Core/UniversalModules/EXFacebook/EXScopedFacebook.m

```diff
@@ -48,15 +48,16 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId andParams:(NSDicti
     BOOL hasPreviouslySetAutoInitEnabled = [_settings boolForKey:AUTO_INIT_KEY];
     BOOL manifestDefinesAutoInitEnabled = [params[@"manifest"][@"facebookAutoInitEnabled"] boolValue];
 
-    NSString *facebookAppId = params[@"manifest"][@"facebookAppId"];
+    NSString *scopedFacebookAppId = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"FacebookAppID"];
+    UMLogInfo(@"Overriding Facebook App ID with the Expo Client's. To test your own Facebook App ID, you'll need to build a standalone app. Refer to our documentation for more info- https://docs.expo.io/versions/latest/sdk/facebook/");
     NSString *facebookDisplayName = params[@"manifest"][@"facebookDisplayName"];
 
     if (hasPreviouslySetAutoInitEnabled || manifestDefinesAutoInitEnabled) {
-      // FacebookAppId is a prerequisite for initialization.
       // This happens even before the app foregrounds, which mimics
       // the mechanism behind EXFacebookAppDelegate.
-      if (facebookAppId) {
-        [FBSDKSettings setAppID:facebookAppId];
+      // Check for FacebookAppId in case this is a custom client build
+      if (scopedFacebookAppId) {
+        [FBSDKSettings setAppID:scopedFacebookAppId];
```

Comment:
Looking from a wider perspective: doesn't Facebook SDK use `Info.plist` as the default source of `FacebookAppID`? If so, do we need to `setAppId:` and `setDisplayName:` manually (just trying to make the code clearer for future maintainers, so they don't scratch their heads why do we set already set value)?

Reply:
I suppose we don't, my thinking is this might make it more obvious to future maintainers that the provided appId is overridden. But with the addition of the `UMLogInfo`, it's already clear what's happening, so good idea to remove setAppId and setDisplayName